src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +56,243 @@ contract DssSpellCollateralAction {
+    uint256 constant ZERO_PCT_RATE           = 1000000000000000000000000000;
+    uint256 constant ZERO_ZERO_FIVE_PCT_RATE = 1000000000015850933588756013;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+    uint256 public constant RAY = 10**27;
+    uint256 public constant RAD = 10**45;
+
+    // -- RWA008 MIP21 components --
+    address constant RWA008                    = 0x30434AA15F85598F45406d08A79dCdD9a79335e9;
+    address constant MCD_JOIN_RWA008_A         = 0x4ce65E856f824C2b4a2eeD88E79b839eB366967d;
+    address constant RWA008_A_URN              = 0x6351915f840937Edba75656727f20165185FaB83;
+    address constant RWA008_A_URN_CLOSE_HELPER = 0xb220461eFFa9c0b1D09047D08Bf116DfbD9814fA;
+    address constant RWA008_A_INPUT_CONDUIT    = 0x6298a2498b4E3789bb8220Cf5f5b44f518509e2a;
+    address constant RWA008_A_OUTPUT_CONDUIT   = 0x1FE95E519F0CE8eCF1cdC885f4DeA7913e146149;
+    // SocGen's wallet
+    address constant RWA008_A_OPERATOR         = 0x3F761335890721752476d4F210A7ad9BEf66fb45;
+    // DIIS Group wallet
+    address constant RWA008_A_MATE             = 0xb9444802F0831A3EB9f90E24EFe5FfA20138d684;
+
+    uint256 constant RWA008_A_INITIAL_DC       = 30_000_000 * RAD;
+    uint256 constant RWA008_A_INITIAL_PRICE    = 30_437_069 * WAD;
+    uint48 constant  RWA008_A_TAU              = 0;
+
+    uint256 constant RWA008_REG_CLASS_RWA      = 3;
+
+    string constant RWA008_DOC                 = "QmdfzY6p5EpkYMN8wcomF2a1GsJbhkPiRQVRYSPfS4NZtB";
+    // -- RWA008 end --
+
+    // -- RWA009 MIP21 components --
+    address constant RWA009                  = 0xd2B101854F64Df738bA601840279838568583F39;
+    address constant MCD_JOIN_RWA009_A       = 0x7122B934F02A15954282Ed41572Ada539864773a;
+    address constant RWA009_A_URN            = 0xD2C8588C72026171Ec3a17369ad0f0734E30915d;
+    address constant RWA009_A_JAR            = 0xa484C16D2Ca15706c4B875710d9e80b7F101572B;
+    // Goerli: CES Goerli Multisig / Mainnet: Genesis
+    address constant RWA009_A_OUTPUT_CONDUIT = 0x7a3D23Dc73F7ead55399597aAE6e525b3DF95A88;
+
+    // MIP21_LIQUIDATION_ORACLE params
+    uint256 constant RWA009_A_INITIAL_DC     = 100_000_000 * RAD;
+    uint256 constant RWA009_A_INITIAL_PRICE  = 100_000_000 * WAD;
+    uint48  constant RWA009_A_TAU            = 0;
+
+    uint256 constant RWA009_REG_CLASS_RWA    = 3;
+
+    string constant RWA009_DOC               = "QmZG31b6iLGGCLGD7ZUn8EDkE9kANPVMcHzEYkvyNWCZpG";
+    // -- RWA009 END --
+
+    function onboardRwa008(
+        ChainlogAbstract CHANGELOG,
+        IlkRegistryAbstract REGISTRY,
+        address MIP21_LIQUIDATION_ORACLE,
+        address MCD_VAT,
+        address MCD_JUG,
+        address MCD_SPOT
+    ) internal {
+        // RWA008-A collateral deploy
+
+        // Set ilk bytes32 variable
+        bytes32 ilk = "RWA008-A";
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA008_A).vat() == MCD_VAT, "join-vat-not-match");
```

Comment:
We could extend the checks for verifying the urn storage values match with the core contracts and the gemJoin itself.

Reply:
It's funny, we're on the same timeline.  I just thought this 5m ago, then manually verified the addresses anyway.

Reply:
It's done!